components/web/php/php-7_3/files/php_config.ini

```diff
@@ -0,0 +1,6 @@
+#if defined(_LP64) || defined (__amd64)
+#include <main/php_config-64.h>
+#else
+#include <main/php_config-32.h>
```

Comment:
Do we still ship any of these two? I see only `main/php_config.h`.

Reply:
No.